runtime/include/chpl-privatization.h

```diff
@@ -27,12 +27,7 @@ void chpl_privatization_init(void);
 
 void chpl_newPrivatizedClass(void*, int64_t);
 
-// Implementation is here for performance: getPrivatizedClass can be called
-// frequently, so putting it in a header allows the backend to fully optimize.
```

Comment:
This was moved in https://github.com/chapel-lang/chapel/pull/6212, and it had a pretty significant impact on performance of prk stencil. We should definetly do some performance analysis of this PR before merging.

Reply:
Unfortunately, its going to result in a minor performance regression in any-case for `chpl_getPrivatizedClass`, and definitely not by too much, and now both `chpl_newPrivatizedClass` (when it doesn't need to allocate more space) and `chpl_clearPrivatizedClass` will be on-par with it... hopefully. This is bleeding-edge stuff, after-all.

Reply:
Overall, I can say that there is one way that will likely counter any regression, but that involves a complete overhaul to privatization as a whole... that'd be an interesting GSoC student project though :)